src/subscription_manager/cli_command/syspurpose.py

```diff
@@ -36,10 +36,9 @@
 
 class SyspurposeCommand(CliCommand):
     """
-    Syspurpose command for generic actions. This command will be used for all
-    syspurpose actions in the future and it will replace addons, role,
-    service-level and usage commands. It will be possible to set service-type
-    using this command.
+    Syspurpose command for generic actions. This command replaced the addons,
+    role, service-level, and usage commands and is used for all syspurpose
+    actions. It will be possible to set service-type using this command.
```

Comment:
if you really want to change this comment in this PR, then simply drop the bit:

| replaced the addons, role, service-level, and usage commands and

once those top-level commands are removed, those references point to removed bits which are hard to find unless you dig into the history; better simply say what the class/command does currently

Reply:
Agreed. Fixed.